client/src/components/ProductDetailsComponents/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.css';
+import ProductTitleAndLogo from '../ProductTitleAndLogo/ProductTitleAndLogo';
+import AddToBasket from '../AddToBasket/AddToBasket';
+
+const propTypes = {
+  productInfoDetails: PropTypes.instanceOf(Object),
+  updateBasket: PropTypes.func,
+};
+
+const defaultProps = {
+  productInfoDetails: null,
+  updateBasket: null,
+};
+
+export default class ProductDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const { productInfoDetails, updateBasket } = this.props;
+    // console.log('PRODUCT DETAILS : ', productInfoDetails);
+    return (
+      <div className={styles.temp}>
+        <div>
+          <h1>ProductDetails</h1>
+          <ProductTitleAndLogo productInfoDetails={productInfoDetails} />
+          <AddToBasket
+            updateBasket={updateBasket}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+ProductDetails.propTypes = propTypes;
```

Comment:
proptypes and defaultprops should probably be next to each other on document.

Reply:
Hi @jonfu86 This is one of the airbnb style guide. Can I keep this way?